lib/katello/tasks/delete_orphaned_content.rake

```diff
@@ -1,7 +1,9 @@
 namespace :katello do
   task :delete_orphaned_content => ["environment"] do
     User.current = User.anonymous_admin
-    Katello::Repository.delete_orphaned_content
+    SmartProxy.with_content.each do |proxy|
+      ForemanTasks.async_task(Actions::Katello::CapsuleContent::RemoveOrphans, :capsule_id => proxy.id)
+    end
```

Comment:
I'd catch an exception within the loop and log the exception.  I'm assuming it would error if the capsule was down